optimize/client/src/components/Home/Home.js

```diff
@@ -61,7 +61,7 @@ export function Home({mightFail, user}) {
       setIsLoading(true);
       setSorting({key: sortBy, order: sortOrder});
       mightFail(
-        loadEntities(sortBy, sortOrder),
+        loadEntities(null, sortBy, sortOrder),
```

Comment:
I do not like this as it is not very readable for someone looking at this the first time.
I suggest creating a function called `loadAllEntities` and in `loadEntities` function we can either call the `loadAllEntities` one or the `loadCollectionEntities` one if a collection is provided.

Reply:
WHYT if we change the order of arguments so the collection ID comes last? This way we have just one function still, but we dont have to pass `null` as the first argument?